### PR TITLE
Add documentation for netatmo public component

### PR DESCRIPTION
**Pull request by colinfrei**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15684

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.netatmo_public.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+sensor:
+  - platform: netatmo_public
+    areas:
+      - lat_ne: 40.719
+        lon_ne: -73.735
+        lat_sw: 40.552
+        lon_sw: -74.105
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.netatmo_public.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — APPROVED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
RFC

**Comment by frenck** [comment]
Hi there @colinfrei, thank you for sending in this PR.
I've reviewed it and left you some feedback. Could you please take a look? 👍

File: source/_components/sensor.netatmo_public.markdown (hunk 1 above)
Comment (colinfrei) [reply source/_components/sensor.netatmo_public.markdown]: Thanks, updated that. Looks like I missed that part in the docs, and the example I looked at wasn't using that tag.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @colinfrei!

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**